src/libs/actions/IOU.ts

```diff
@@ -1968,7 +1967,7 @@ function getTrackExpenseInformation(
     moneyRequestReportID = '',
     linkedTrackedExpenseReportAction?: OnyxTypes.ReportAction,
     existingTransactionID?: string,
-): TrackExpenseInformation | EmptyObject {
+): OnyxEntry<TrackExpenseInformation> {
```

Comment:
```suggestion
): TrackExpenseInformation | null {
```

`TrackExpenseInformation` is not a value stored in Onyx